admin/listPurchases/listView.html

```diff
@@ -167,6 +171,40 @@
     }).join("");
 }
 
+function badgeImageList(entry) {
+    let images = [];
+    const badgeNameImageMapping = {
+        "10-minutters-tårn": "10-min-taarn",
+        "12 x telt": "12-x-telt",
+        "200 km mærket": "200km",
+        "36 timer": "36-timer",
+        "Gift med en spejder": "gift-med-en-spejder",
+        "Kystvogter": "kystvogter",
+        "Leder&shy;challenge": "lederchallenge",
+        "Madtropperne": "madtropperne",
+        "Mærkedesigner": "maerkedesigner",
+        "One night stand": "one-night-stand",
+        "Overlevelsestur": "overlevelsestur",
+        "Rollespejd Bronze": "rollespejd-bronze",
+        "Rollespejd Sølv": "rollespejd-soelv",
+        "Rollespejd Guld": "rollespejd-guld",
+        "Skirty mærke": "skirty",
+        "Skirty365 mærke": "skirty365",
+        "Skovskider": "skovskider",
+        "Sleep-over": "sleep-over",
+        "Sovepose­365": "sovepose365",
+        "Spejd­365": "spejd­365",
+        "St. Georgsdag": "st-george",
+        "Voer­lade­gård Hjemme­spejd": "voerladegaard-hjemmespejd",
```

Comment:
Instead of hardcoding this, use the data we already get from maerkelex requests - this should be passed to the viewModel